File: Cpp/Unit01/Exercise04/Program/exercise04.cpp
#include<iostream>
#include<fstream>

int main() {
    // Creating a variable to hold the number
    float num;

    // reading in the data
    std::ifstream indata;
    indata.open("../Unit01/Exercise04/Inputs/input.txt");
    indata>>num;

    // writing out the data
    std::ofstream outdata;
    outdata.open("../Unit01/Exercise04/Outputs/output.txt");
    outdata<<num*num;
    outdata.close();

    return 0;
}

File: Fortran90/Unit01/Exercise04/Program/exercise04.f90
program exercise04
    implicit none

    real :: num

    ! Reading in the input
    open(1, file="../Unit01/Exercise04/Inputs/input.txt")
    read(1,*) num
    close(1)

    ! Writing out the square
    open(1, file="../Unit01/Exercise04/Outputs/output.txt")
    write(1,*) num*num
    close(1)
    
end program exercise04